src/app/collective-rewards/components/TotalRewardsDistributed/TotalRewardsDistributed.tsx

```diff
@@ -0,0 +1,68 @@
+import { Metric } from '@/components/Metric/Metric'
+import { TokenAmountDisplay } from '@/components/TokenAmountDisplay'
+import { CommonComponentProps } from '@/components/commonProps'
+import { usePricesContext } from '@/shared/context/PricesContext'
+import { getTokens, TokenInfo } from '@/lib/tokens'
+import { useGetGaugesArray } from '@/app/collective-rewards/user'
+import { Address } from 'viem'
+import { formatSymbol, getFiatAmount, useGetGaugesNotifyReward } from '@/app/collective-rewards/rewards'
+import { useHandleErrors } from '@/app/collective-rewards/utils'
+import { formatCurrency } from '@/lib/utils'
+import { LoadingSpinner } from '@/components/LoadingSpinner'
+
+interface TokenRewardsProps {
+  gauges: Address[]
+  token: TokenInfo
+}
+
+const TokenRewards = ({ gauges, token: { address, symbol } }: TokenRewardsProps) => {
+  const { prices } = usePricesContext()
+  const { data: rewardsData, isLoading, error } = useGetGaugesNotifyReward(gauges, address)
+
+  useHandleErrors({ error, title: `Error loading rewards for ${symbol}` })
+
+  if (isLoading) {
```

Comment:
Do we have a strategy for loadings?

Reply:
@franciscotobar do you mean from a design perspective? if so, it was never mentioned, so I guess we don't, but let me ask Vlad